core/aws_ddk_core/stages/glue_transform.py

```diff
@@ -155,9 +155,7 @@ def __init__(
             },
             iam_resources=[crawler_arn],
         )
-        crawl_object.add_catch(
-            Fail(self, "failure", cause="Glue Crawler Failed"), errors=["Glue.CrawlerRunningException"]
-        )
+        crawl_object.add_catch(Succeed(self, "crawler-pass"), errors=["Glue.CrawlerRunningException"])
```

Comment:
I think when this gets translate to CDK, it will result in the step function having two different success states. One will be called `success` (defined on line 164) and the other will be called `crawler-pass`, which will only be triggered when the `CrawlerRunningException` is caught. I think this is going to be somewhat confusing for customers.

I would suggest moving the `Succeed(self, "success")` from line 164 to a separate variable, which we then invoke both here and in line 164. That way the state machine will have just one success state, called `"success"`.

Reply:
Good call didn't think about that, thanks